src/AdminConsole/appsettings.json

```diff
@@ -48,6 +49,7 @@
         "Features": {
           "EventLoggingIsEnabled": false,
           "EventLoggingRetentionPeriod": 0,
+          "MagicLinkEmailMonthlyQuota": 1000,
           "MaxAdmins": 1000,
           "MaxApplications": 1000
         },
```

Comment:
Missing quota for enterprise, set to same as Pro.
issue: Let's lower free emails to 100.

Reply:
Done